app/controllers/users_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update_attributes(user_params)
+      flash[:success] = 'User updated!'
+      redirect_to user_path
+    else
+      render action: :edit
+    end
+  end
+
+  def show; end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.
Please remove